DQM/HcalTasks/plugins/DigiTask.cc

```diff
@@ -95,15 +97,15 @@ DigiTask::DigiTask(edm::ParameterSet const& ps):
 		new hcaldqm::quantity::ValueQuantity(hcaldqm::quantity::fQIE10ADC_256),
 		new hcaldqm::quantity::ValueQuantity(hcaldqm::quantity::fN, true),0);
 	_cfC_SubdetPM_QIE1011.initialize(_name, "fC", hcaldqm::hashfunctions::fSubdetPM,
-		new hcaldqm::quantity::ValueQuantity(hcaldqm::quantity::fQIE10fC_10000),
+		new hcaldqm::quantity::ValueQuantity(hcaldqm::quantity::fQIE10fC_400000),
```

Comment:
How are these allocations cleared?

Reply:
These are actually not cleared... I think this method of booking histograms (established by Viktor a few years ago?) was considered OK because it happens in `beginRun`. I'd be happy to consider patching this up, but I would prefer that be done independently at a later time.

Reply:
actually not - the initial (well, initial fixed) design cleaned these sort of allocations (go look back at the history to see the discussion...)

Reply:
Assumption that beginRun happens once is wrong. It may happen multiple times. Not in the central workflows, but it does happen.

`But I would prefer that be done independently at a later time.` 
I would prefer if it was done now. `later` usually means never.

Reply:
As pointed out by @davidlange6 , the ValueQuantity objects are deleted here: https://github.com/cms-sw/cmssw/blob/54c17f0c102be607ddab88b6a5edcd104c0a81f7/DQM/HcalCommon/src/Container1D.cc#L22. So I think there is no problem.

Reply:
Ah, great then!